tileview/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -772,7 +772,9 @@ public void onPanEnd( int x, int y, Origination origin ) {
 
   @Override
   public void onZoomBegin( float scale, Origination origin ) {
-    mDetailLevelManager.lockDetailLevel();
+    if( !Origination.DOUBLE_TAP.equals( origin ) ) {
```

Comment:
> if we allow rendering while panning, why should we forbid rendering while zooming?

a couple important points: 1, we allow render _optionally_, and it is not default behavior, the user must opt-in to render while panning, while this change affects all users and all situations.  2., `lockDetailLevel` does not prevent rendering, it prevents the detail level from changing.